certbot/certbot/display/ops.py

```diff
@@ -19,58 +19,31 @@
 logger = logging.getLogger(__name__)
 
 
-def get_email(invalid: bool = False, optional: bool = True) -> str:
+def get_email(invalid: bool = False) -> Optional[str]:
     """Prompt for valid email address.
 
     :param bool invalid: True if an invalid address was provided by the user
-    :param bool optional: True if the user can use
-        --register-unsafely-without-email to avoid providing an e-mail
 
     :returns: e-mail address
     :rtype: str
 
     :raises errors.Error: if the user cancels
 
     """
-    invalid_prefix = "There seem to be problems with that address. "
-    msg = "Enter email address (used for urgent renewal and security notices)\n"
-    unsafe_suggestion = ("\n\nIf you really want to skip this, you can run "
-                         "the client with --register-unsafely-without-email "
-                         "but you will then be unable to receive notice about "
-                         "impending expiration or revocation of your "
-                         "certificates or problems with your Certbot "
-                         "installation that will lead to failure to renew.\n\n")
-    if optional:
-        if invalid:
-            msg += unsafe_suggestion
-            suggest_unsafe = False
-        else:
-            suggest_unsafe = True
-    else:
-        suggest_unsafe = False
+    msg = "Enter email address or hit Enter to skip.\n"
+    if invalid:
+        msg = "The server reported a problem with your email address. " + msg
```

Comment:
We probably only want to show this addition the first loop. That is, 1) server reports a problem 2) we tell them that and ask for email again 3) they enter a bad email 4) we tell them that the new email is bad, not re-tell them about the server email problem